project.go

```diff
@@ -107,14 +106,6 @@ func (s *projects) List(ctx context.Context, organization string, options *Proje
 	return p, nil
 }
 
-func (o *ProjectListOptions) valid() error {
-	if o == nil || o.PageNumber == 0 || o.PageSize == 0 {
-		return ErrInvalidPagination
-	}
-
-	return nil
-}
-
```

Comment:
Curious: Why remove the validation?

Reply:
The ProjectListOptions should be optional and not required. This validation checks that it is always present so removed it.

Reply:
Ah! I think I understand. In that case you can remove the nil check here. A "nil ProjectListOptions" is valid.

Reply:
```go
func (o *ProjectListOptions) valid() error {
	if o != nil && (o.PageNumber == 0 || o.PageSize == 0) {
		return ErrInvalidPagination
	}

	return nil
}
```

Reply:
That makes sense. Thanks @brandonc!